server/index.mjs

```diff
@@ -0,0 +1,216 @@
+import express from 'express';
+import multer from 'multer';
+import cors from 'cors';
+import fs from 'fs';
+import XLSX from 'xlsx';
+import PizZip from 'pizzip';
+import Docxtemplater from 'docxtemplater';
+import serverless from 'serverless-http';
+import bodyParser from "body-parser";
+import {v4 as uuidv4} from 'uuid';
+import archiver from 'archiver';
+
+const app = express();
+
+app.use(cors());
+app.use(express.json());
+app.use(bodyParser.urlencoded({extended: false}));
+
+const storage = multer.memoryStorage();
+const upload = multer({storage: storage});
+
+const generateData = async (input, fileNameTemplateDocx, folderName) => {
```

Comment:
I would put these functions in a services folder and then just call them from `index.mjs`,
same with generateViaXLSX etc